### PR TITLE
Add libcups

### PR DESCRIPTION
**Pull request by isuruf**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/libcups/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: f58010813fd6903f690cdb0c0b91e4d1bc9e5b9570c28734229ba3ed2908b76c
+
+build:
+  number: 0
+  skip: True   # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/libcups/meta.yaml]:
```suggestion
    - {{ compiler('cxx') }}
    - make
```

File: recipes/libcups/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/apple/cups/releases/download/v{{ version }}/cups-{{ version }}-source.tar.gz
+  sha256: f58010813fd6903f690cdb0c0b91e4d1bc9e5b9570c28734229ba3ed2908b76c
+
+build:
+  number: 0
+  skip: True   # [not linux]
```
Comment (chrisburr) [inline recipes/libcups/meta.yaml]: Have you tried also building this for macOS?
```suggestion
  skip: True   # [not unix]
```

Comment (isuruf) [reply recipes/libcups/meta.yaml]: I have and it doesn't build. This library is part of MacOS SDK anyway

**Comment by isuruf** [comment]
Thanks for the review and merge

**Pull request merged**